lti_consumer/lti.py

```diff
@@ -159,6 +159,32 @@ def get_signed_lti_parameters(self):
                 if len(language_preference) == 1:
                     self.xblock.user_language = language_preference[0].value
 
+            credit_service = self.xblock.runtime.service(self.xblock, 'credit')
+            user_id = getattr(real_user_object, "id", None)
+            if self.xblock.transmit_course_mode_and_status and credit_service and user_id:
```

Comment:
Got it. I'd say add what ever you think would be more helpful than unhelpful. Thanks.